test/replication/qsync_basic.result

```diff
@@ -409,6 +421,9 @@ test_run:switch('replica')
  | ---
  | - true
  | ...
+test_run:wait_lsn('replica', 'default')
+ | ---
+ | ...
 box.space.sync:select{9}
  | ---
  | - []
```

Comment:
AFAIR, this is the place we talked about (regarding `Surprising, there was a place where replica expected
to have data from master, that master rolls back.`)

Seems it's not here, so let's not mention it in the commit mesage.

Reply:
Yep, you're right. At some point the following select from the test space resulted in [9]. Perhaps it was because of intermittent changes with a triggering change in relay. Removing.